cpp/tests/lists/sort_lists_tests.cpp

```diff
@@ -198,22 +182,20 @@ TEST_F(SortListsInt, NullRows)
   LCW<T> l{{{1, 2, 3}, {4, 5, 6}, {7}}, valids.begin()};  // offset 0, 0, 3, 3
 
   auto const [sorted_lists, stable_sorted_lists] =
-    generate_sorted_lists(lists_column_view{l}, {}, {});
+    generate_sorted_lists(cudf::lists_column_view{l}, {}, {});
   CUDF_TEST_EXPECT_COLUMNS_EQUAL(sorted_lists->view(), l);
   CUDF_TEST_EXPECT_COLUMNS_EQUAL(stable_sorted_lists->view(), l);
 }
 
-/*
 // Disabling this test.
 // Reason: After this exception "cudaErrorAssert device-side assert triggered", further tests fail
-TEST_F(SortListsInt, Depth)
+TEST_F(SortListsInt, DISABLED_Depth)
```

Comment:
Is this `DISABLED_` prefix recognized by GTest? What's the magic / reason for change here? Does this test still cause real problems?

Reply:
The `DISABLED_` prefix is indeed gtest [magic](https://github.com/google/googletest/blob/main/docs/advanced.md#temporarily-disabling-tests).
The code was previously commented out and the reason is commented on line 208.
I opted to disable the test so the code is verified by the compiler.